### PR TITLE
chore(ci): Add job to update database for testrail dashboards.

### PR DESCRIPTION
**Pull request by b4handjr**
## Description

- [✅] This PR conforms to the [Contribution Guidelines](/CONTRIBUTING)
- [ ] AI tools were used in generating this pull request

## Issues

Closes: # [ECTEEN-130](https://mozilla-hub.atlassian.net/browse/ECTEEN-130)

[ECTEEN-130]: https://mozilla-hub.atlassian.net/browse/ECTEEN-130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by b4handjr** [comment]
[Action Run](https://github.com/mozilla/ecosystem-test-scripts/actions/runs/12167673296)

File: .github/workflows/production_testrail_metrics_weekly.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        run: |
+            echo "CLOUD_SQL_DATABASE_USERNAME=${{ secrets.CLOUD_SQL_DATABASE_USERNAME }}" >> $GITHUB_ENV
+            echo "CLOUD_SQL_DATABASE_PASSWORD=${{ secrets.CLOUD_SQL_DATABASE_PASSWORD }}" >> $GITHUB_ENV
+            echo "CLOUD_SQL_DATABASE_NAME=stage-test-metrics" >> $GITHUB_ENV
+            echo "CLOUD_SQL_DATABASE_PORT=${{ secrets.CLOUD_SQL_DATABASE_PORT }}" >> $GITHUB_ENV
+            echo "TESTRAIL_HOST=${{ secrets.TESTRAIL_HOST }}" >> $GITHUB_ENV
+            echo "TESTRAIL_USERNAME=${{ secrets.TESTRAIL_USERNAME }}" >> $GITHUB_ENV
+            echo "TESTRAIL_PASSWORD=${{ secrets.TESTRAIL_PASSWORD }}" >> $GITHUB_ENV
+      - name: Update DB - test coverage
+        run: python ./__main__.py --report-type test-case-coverage --platform ecosystem --project ${{ vars.ECOSYSTEM_PROJECTS }}
```
[the remark this answers is not in the document]
Comment (pdehaan) [reply .github/workflows/production_testrail_metrics_weekly.yml:44]: Ah, sorry, should have re-read the PR and followed the threads. Me thinks it's this `__main__.py` file: https://github.com/mozilla-mobile/testops-dashboard/blob/master/__main__.py

File: .github/workflows/production_testrail_metrics_weekly.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          credentials_json: ${{ secrets.GCLOUD_AUTH }}
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+      - name: Enable cloud SQL proxy to temp instance
+        run: |
+          curl -o cloud-sql-proxy https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.7.0/cloud-sql-proxy.linux.amd64
+          chmod +x cloud-sql-proxy
+          nohup ./cloud-sql-proxy ${{ secrets.CLOUD_SQL_CONNECTION_NAME }} > cloud-sql-proxy.out 2> cloud-sql-proxy.err < /dev/null &
+          echo $!
+          echo CLOUD_SQL_PROXY_PID=$(echo $!) >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (b4handjr) [reply .github/workflows/production_testrail_metrics_weekly.yml:31]: Not sure. Doing research on how to do this on github, this is how everyone was doing it. Maybe it's just a reference to show it is running

**Review by pdehaan — APPROVED** [review]
R+ to unblck in case you want to merge to test out Cronning.

**Pull request merged**